src/bidManager.js

```diff
@@ -203,13 +183,33 @@ class BidManager {
       bidResponse.pixelsRendered = true;
     }
   }
-}
 
-BidManager.SERVER_MAP = {
-  EU: 'adserver.adtech.de',
-  US: 'adserver.adtechus.com',
-  Asia: 'adserver.adtechjp.com'
-};
+  resolveBidRequest(bidRequestConfig, placementConfig) {
+    if (bidRequestConfig.dcn && placementConfig.pos) {
+      return new NexageGetBidRequest(bidRequestConfig, placementConfig);
+    } else if (bidRequestConfig.network && placementConfig.placement) {
+      return new MarketplaceBidRequest(bidRequestConfig, placementConfig);
+    } if (this.isNexagePostRequest(placementConfig.openRtbParams)) {
+      return new NexagePostBidRequest(bidRequestConfig, placementConfig);
+    }
+  }
+
+  isNexagePostRequest(openRtbParams) {
```

Comment:
You do this much more elegantly than me 👍 . Might copy this to be the same in the AOL Container Tag.